Library/Formula/eg.rb

```diff
@@ -0,0 +1,83 @@
+class Eg < Formula
+  desc "Expert Guide. Norton Guide Reader For GNU/Linux"
+  homepage "http://www.davep.org/norton-guides/"
+  url "http://www.davep.org/norton-guides/eg-1.00.tar.gz"
+  sha256 "e985b2abd160c5f65bea661de800f0a83f0bfbaca54e5cbdc2e738dfbbdb164e"
+
+  depends_on "s-lang"
+
+  patch :DATA
+
+  def install
+    inreplace %w[
+      eg.c
+      egcmplte.c
+      egdir.c
+      egdraw.c
+      eggetfld.c
+      eghelp.c
+      eglib.c
+      egmenu.c
+      egnavgte.c
+      egregex.c
+      egscreen.c
+      egsigs.c], "<slang/slang.h>", "<slang.h>"
+    inreplace "eglib.c", "/usr/share/", "#{HOMEBREW_PREFIX}/share/"
```

Comment:
Likewise, this is something we should be able to set in the `make` stage rather than inreplacing every time, so ideally poke upstream on this as well.